docs/en_US/query_tool_toolbar.rst

```diff
@@ -1,231 +1,234 @@
+   | *Delete*             | Click the *Delete* icon to mark the selected rows for deletion. These marked rows get deleted     | Option/Alt +   |
+   |                      |                                                                                                   | Shift + D      |
+   |                      | when you click the *Save Data Changes* icon.                                                      |                |
+   +----------------------+---------------------------------------------------------------------------------------------------+----------------+
+   | *Save Data Changes*  | Click the *Save Data Changes* icon to save data changes (insert, update, or delete) in the Data   | F6             |
+   |                      | Output Panel to the server.                                                                       |                |
+   +----------------------+---------------------------------------------------------------------------------------------------+----------------+
+   | *Save results to*    | Click the Save results to file icon to save the result set of the current query as a delimited    | F8             |
+   | *file*               | text file (CSV, if the field separator is set to a comma). This button will only be enabled when  |                |
+   |                      | a query has been executed and there are results in the data grid. You can specify the CSV/TXT     |                |
+   |                      | settings in the Preference Dialogue under SQL Editor -> CSV/TXT output.                           |                |
+   +----------------------+---------------------------------------------------------------------------------------------------+----------------+
+   | Graph Visualiser     | Use the Graph Visualiser button to generate graphs of the query results.                          |                |
+   +----------------------+---------------------------------------------------------------------------------------------------+----------------+
+   | SQL                  | Use the SQL button to check the current query that gave the data.                                 |                |
+   +----------------------+---------------------------------------------------------------------------------------------------+----------------+
+
+Status Bar
+**********
+
+.. image:: images/query_status_bar.png
+    :alt: Query tool status bar
+    :align: center
+
+The status bar shows the following information:
+
+* **Total rows**: The total number of rows returned by the query.
+* **Query complete**: The time is taken by the query to complete.
+* **Rows selected**: The number of rows selected in the data output panel.
+* **Changes staged**: This information showed the number of rows added, deleted, and updated.
+* **LF/CRLF**: This information showed the end of line sequence of text based on operating system.

```

Comment:
```
It shows the end of line sequence to be used for the editor. When opening an empty editor, it will be decided based on OS.  And when opening an existing file, it will be based on file end of lines. One can change the EOL by clicking on any of the options.
```

Remove your text.